### ISSUE TITLE
Can't seek videos using mouse "4k mode"

### ISSUE THREAD

**Issue by Julia91313** [yuliskov/SmartYouTubeTV#192]
I have opened the app in 1080 mode and 4k mode and have noticed that while in 1080 i am able to scroll through the video but while watching a video in the 4k mode I am unable to scroll through it. Every time i click to fast forward the video it ends up bringing me back to my current location in the video.

**Comment by yuliskov** [comment]
@Julia91313 sorry, but I didn't understand what you talking about. Could you explain the problem more deeply?

**Comment by Julia91313** [comment]
For example, I want to watch a part of video that is 8 minutes long. The problem is I don't want to watch the full video, but I would like to start watching from 5 minutes in. In 1080 mode I am able to scroll through the video and start watching 5 minutes into it. With 4k mode, I am unable to do that. It only allows me to watch the video from the location (time) that I am currently at.

**Comment by yuliskov** [comment]
@Julia91313 are you unable to seek through the video in the 4K mode? Right?

**Comment by Julia91313** [comment]
Yes, I am unable to seek through the video in 4K mode

**Comment by Julia91313** [comment]
I click with my mouse on the seek bar and on the screen i get "loading...", but then nothing happens. I click to go backwards and I get the same message. I have tried clicking the circle on the seek bar and dragging it forward and backwards but nothing happens then either. Whether I try seeking forwards or backwards with this method the video remains at the same location. I use these same methods in 1080 mode and it works perfectly.

I have just discovered that the only way I can seek through the video in 4K mode is to click on the left and right arrows of my remote control, which seeks 10 seconds at a time.
